deploy/helm/sumologic/values.yaml

```diff
@@ -4302,6 +4302,34 @@ otellogs:
     #     mountPath: /certs
     #     readOnly: true
 
+  ## additionalDaemonSets allows to set daemonsets with affinity, nodeSelector and resources
+  ## different than the main DaemonSet
+  ## Be careful and set nodeAffinity for the main DaemonSet,
+  ## as we do not support multiple pods of otellogs on the same node
```

Comment:
@SumoLogic/open-source-collection-team 

Do you think it is safe assumption that if customer needs to set it, thew will know how to do it properly?
Or could we limit to `nodeSelector` and figure out `nodeAffinities` by ourselves?

Reply:
Let them figure it out imo, they should know what they're doing if they need to use this.